src/scripts/origins.json

```diff
@@ -277,6 +277,11 @@
     "name": "OnlyOffice",
     "clone": "true"
   },
+  "osticket.com": {
+    "url": "*://*supportsystem.com/*",
```

Comment:
Ah it appears it the host part needs to have a dot between the wildcard and the domain `*.supportsystem.com`